drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -186,7 +183,7 @@ def studio_view(self, context):
 
         js_templates = loader.load_unicode('/templates/html/js_templates.html')
         help_texts = {
-            field_name: self._(field.help)
+            field_name: self.ugettext(field.help)
```

Comment:
@asadiqbal08 @mattdrayer 

I used Poedit and the `text.po` file for English to create `text.po` and `text.mo` files for French with a few translations for help texts, and put them in `/edx/app/edxapp/xblock-drag-and-drop-v2/drag_and_drop_v2/translations/fr/LC_MESSAGES/` (this is where DnDv2 is mounted on my Devstack). Then I started Studio and accessed the EDIT dialog of a unit containing a DnDv2 block. The strings didn't seem to get translated, they were still displayed in English. (Studio itself is being translated correctly.)

If I'm missing something, could you please post detailed instructions for testing this PR in Studio?